LibraryManagementSystem/Forms/DeleteMemberTab.cs

```diff
@@ -29,18 +29,18 @@ private void btnDelete_Click(object sender, EventArgs e)
             }
             else
             {
-                con = new SqlConnection("Data Source=" + Program.globalServer + "\\SQLEXPRESS;Initial Catalog=libraryData;Integrated Security=True");
+                con = new SqlConnection("Data Source=" + Program.globalServer + "\\SQLEXPRESS;Initial Catalog=LibDat;Integrated Security=True");
                 con.Open();
                 int memberID = Convert.ToInt32(txtMemberID.Text);
                 bool idExist = false;
 
-                cmd = new SqlCommand("SELECT * FROM members_data", con);
+                cmd = new SqlCommand("SELECT * FROM member", con);
 
                 // Selects value from members_data table in the database with the same memberID as inputted and assigns them to their respective variables
```

Comment:
Comment needs to be updated here to reflect new table name:
<kbd>// Selects value from member table in the database with the same mem_id as inputted and assigns them to their respective variables</kbd>